sdk/component/conf.go

```diff
@@ -248,10 +249,10 @@ func newRunnerConfig() (*RunnerConfig, error) {
 
 		conf.storerConfig.storeType = storageType
 
-		dbDSN, err := fromEnvOrDefault(
+		dbDSN, err := env.GetOrDefault(
 			envVarKeyBackendStoreDSN,
 			"smithy.db",
```

Comment:
Should we put all defaults into constants at the top of the file or somewhere cenralized?

Reply:
Right now this is only in use here and we don't have other default vals for env vars so I think it would be more consistent as is

I agree that we can do what you suggest if we add more of them :)